student_class.rb

```diff
@@ -0,0 +1,12 @@
+require './person_class'
```

Comment:
### Optional
- [ ] Let us use `require_relative` when working with files in our directory